[PLAYER-84] adding accesToken generation to readme (#149)
Genymobile/genymotion-device-web-player: README.md

</script>
```

### 🔑 A word about the token

#### Saas

> To connect a [SaaS](https://docs.genymotion.com/saas/) instance you need a valid token. This token must be generated by using your [endpoint API](https://developer.genymotion.com/saas/#tag/Instances-v1/operation/accessToken). When using this API, don't forget to include the x-api-token HTTP header with a valid [Bearer](https://developer.genymotion.com/saas/#tag/Users-v1/operation/login) or [APIToken](https://www.genymotion.com/blog/api-tokens-for-genymotion-saas/)

Reviewer: It may not be very clear for people who are not very familiar. I thing you should make it clear that the generated token is _not_ the same token as the API token.
Maybe something like this:
```suggestion
To connect a [SaaS](https://docs.genymotion.com/saas/) instance you need to generate and use a specific token. This must be done by using our [endpoint API](https://developer.genymotion.com/saas/#tag/Instances-v1/operation/accessToken). When using this API, don't forget to include the x-api-token HTTP header with a valid [Bearer](https://developer.genymotion.com/saas/#tag/Users-v1/operation/login) or [API Token](https://www.genymotion.com/blog/api-tokens-for-genymotion-saas/)
```

Reviewer: I also think you should add a code example here as well.
